### ISSUE TITLE
Delete offers tab

### ISSUE THREAD

**Issue by sarahsporck** [digitalfabrik/integreat-app#2467]
## New description

**Is your feature request related to a problem? Please describe.**
We want to remove the offers tab and move all of them to embedded offers.

**Describe the solution you'd like:**
Remove all offer specific routes, pages, helpers, ... that are not needed for embedded offers.
Make sure embedded offers continue to work.
Remove screenshots, intro slide, ...

**Testing information**
Make sure embedded offers continue to work.

## Old description
As a first step:
- Offers should be moved to local information. That means:  If there is no "Arbeit" 1st level category all the current offers (as they are all work related) should be childs of that 1st level "Arbeit category)
- If there already is an "Arbeit" 1st level category, a 2nd level category "Offers" should be added as a child which has all the current offers as child.
- In the offers tab, there should be a hint that offers where moved including an internal link to the 1st or 2nd level category where the offers are now shown.

Acceptance criteria:
- Offline usage is handled on native (either offers are saved offline as well or at least a good error is displayed)
- Hint shown in the offers tab
- Offers as child of 1st or 2nd level category
- Delete offers screenshot from store metadata

Things to be discussed/unclear:
- If page based offers are nearly ready, does it really make sense to do this first and than change it a short time later again?
- How can we now in the offers if we have to link to a 1st or 2nd level category?
- How do we know in other languages than German whether there is a 1st level category "Arbeit"? Do we check depending on the title or the path or something else?
- How to handle offline usage?
- Can we find the offers category from the search?
- How does feedback work?

In the future we will remove the offers navigation item and change to page based offers (#2313).

**Comment by steffenkleinle** [comment]
I think this is blocked by #2313 right?

**Comment by sarahsporck** [comment]
Hmm. Don't really think so. We discussed this at the conference. And our first decision was to just move all the offers into one Root Category, I think. @ztefanie do you remember?

**Comment by ztefanie** [comment]
Yes I also remember this was discussed and also @dkehne agreed on this?

**Comment by steffenkleinle** [comment]
I adjusted the description with things that are unclear here/not that easy imo. I think this is not worth it if we are switching to page based offers with support from the cms and a new design anyway...

**Comment by ztefanie** [comment]
Please also delete the offers-tab screenshots for the store presence (google and apple)
